Cargo.toml

```diff
@@ -2,8 +2,16 @@
 name = "consensus"
 version = "0.1.0"
 edition = "2021"
+repository = "https://github.com/dusk-network/consensus"
+description = "An implementation of Succinct Attestation consensus protocol"
+categories = ["cryptography", "cryptography::cryptocurrencies"]
+keywords = ["consensus", "dusk", "cryptocurrency", "blockchain"]
+license = "MPL-2.0"
+exclude = [".github/*", ".gitignore", ".env", ".vscode"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "consensus"
```

Comment:
This crate name has been already taken on crates.io

Reply:
Good point! Any suggestions? E.g `dusk-consensus` ?

Reply:
Yes, I'm for that